### PR TITLE
Add docstring to reexec.Command

### PR DESCRIPTION
**Pull request by LK4D4**

### CODE REVIEW

**Comment by estesp** [comment]
LGTM

File: pkg/reexec/command_windows.go [hunk 1]
```diff
@@ -6,6 +6,9 @@ import (
 	"os/exec"
 )
 
+// Command returns *exec.Cmd which have Path as current binary. Also it setting
```
Comment (runcom) [inline pkg/reexec/command_windows.go]: `Also it setting` could be removed here for windows right?

**Comment by duglin** [comment]
Aside from small typo found by @runcom, LGTM

**Comment by LK4D4** [comment]
@runcom @duglin fixed

**Comment by runcom** [comment]
LGTM waiting janky

**Pull request merged**